packages/node_modules/@ciscospark/react-container-read-receipts/src/selectors.js

```diff
@@ -16,25 +17,28 @@ export const getReadReceipts = createSelector(
     const readParticipants = participants
     .filter((participant) =>
       participant.get(`id`) !== currentUser.id &&
-      participant.getIn([`roomProperties`, `lastSeenActivityUUID`]) === activity.id);
+      participant.getIn([`roomProperties`, `lastSeenActivityUUID`]) === activity.id)
+    .toJS();
 
-    const limitParticipants = readParticipants.slice(0, 10)
+    const mappedParticipants = readParticipants
```

Comment:
Would it be easier to add a `.sortBy` to the immutable object instead of converting the full object into a js object and then processing it? Wouldn't need lodash at that point.

Reply:
I've got some more stuff coming that benefits from converting it to a JS object so I'm going to stick with lodash